ob-pattern-5/README.md

```diff
@@ -4,6 +4,93 @@ Resources for building a Helm chart for deployment of WSO2 Open Banking API Mana
 support
 ![WSO2 Open Banking pattern 5 deployment](https://raw.githubusercontent.com/wso2/kubernetes-open-banking/v3.0.0.1/ob-pattern-5/pattern-5.png)
 
+Note : 
+  - UK toolkit is used as a reference toolkit throughout the documentation and in the sample helm chart.
+  - Accelerator docker images are available at [WSO2 Private Docker Registry](https://docker.wso2.com/) and UK toolkit docker images need to be built using the accelerator docker images with the required customizations.
+    - Dockerfile for [wso2-obiam-toolkit-uk](https://github.com/wso2/docker-open-banking/blob/master/dockerfiles/alpine/obiam-uk/Dockerfile)
```

Comment:
these are sample ones right? Shall we mentioned it

Reply:
Fixed in [55ba3cd](https://github.com/wso2/kubernetes-open-banking/pull/81/commits/55ba3cda74dae28b86a408cff988298cfbe96ca6)